.github/workflows/run-acceptance-tests.yml

```diff
@@ -140,6 +140,7 @@ jobs:
       with:
         name: pulumi-${{ env.PROVIDER }}-provider.tar.gz
         path: ${{ github.workspace }}/bin/provider.tar.gz
+        retention-days: 30
```

Comment:
OOC: why do we upload acceptance test builds to artifactory?

Reply:
excellent question! we download them later in the Workflow to use for actually running the tests (~ L#215 I think).